### DISCUSSION TITLE
Track cursor onParent

### DISCUSSION THREAD

**Discussion by PixelJanitor** [mkosir/react-parallax-tilt#30] [Ideas]
The ability to `trackOnWindow` is great, and close to what we need, but better yet it would be awesome to be able to reference some sort of parent element that enables cursor tilt following when the cursor enters that space and not track based on the entire window at all times.

Example: If there is an entire section of a website that you scroll to and want a specific element inside that section to be tilt-enabled only when your cursor is within that section (not just when within the tilt-able container itself like the default behavior)

**Comment by AnweshGangula** [comment]
I think this feature is implemented in [Vanilla-tilt.js](https://micku7zu.github.io/vanilla-tilt.js/) called - [mouse-event-element](https://micku7zu.github.io/vanilla-tilt.js/#:~:text=inner%20by%20yourself%0A%20%20%20%20%22-,mouse-event-element,-%22%3A%20%20null%2C%20%20%20%20//%20css-selector). 

Is it possible to get this implemented here?
